### ISSUE TITLE
Add a license

### ISSUE THREAD

**Issue by Addisonbean** [i-mint/midnight#5]
I love the themes! I was planning on creating a package on the AUR for this, however there's no license. Would you be able to create a license so it can be added to package managers? Thanks!

**Comment by i-mint** [comment]
Yes, of course I will add a license to the Midnight GTK themes, thanks! for the appreciation
